.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11.0-beta - 3.11.0"]
```

Comment:
I'm curious about the 3.11 version identifier.   I would have expected one or the other but not both separated by " - ".  Is this saying "Prefer 3.11.0, but if not found, use 3.11.0-beta"?

Reply:
"[hyphen ranges](https://github.com/npm/node-semver#hyphen-ranges-xyz---abc) to download and set up the latest available version of Python (includes both pre-release and stable versions)".  https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input